unison-cli/src/Unison/Codebase/Editor/HandleInput/ProjectCreate.hs

```diff
@@ -183,53 +183,151 @@ generateRandomProjectNames = do
+          "brilliant",
+          "courageous",
           "charming",
+          "dazzling",
           "delightful",
+          "determined",
+          "devoted",
+          "elegant",
+          "enchanting",
+          "energetic",
+          "engaging",
           "excited",
+          "fantastic",
+          "fortuitous",
           "friendly",
           "gentle",
           "helpful",
+          "heartwarming",
+          "hilarious",
+          "humorous",
+          "incredible",
+          "imaginative",
           "innocent",
+          "insightful",
           "jolly",
+          "joyous",
           "kind",
           "lucky",
           "magnificent",
+          "marvelous",
           "nice",
           "outstanding",
+          "patient",
+          "philosophical",
           "pleasant",
+          "proficient",
           "quiet",
+          "relaxed",
+          "resourceful",
           "responsible",
           "silly",
+          "sincere",
+          "sensible",
+          "sparkling",
+          "spectacular",
+          "spellbinding",
+          "stellar",
           "thoughtful",
           "useful",
-          "witty"
+          "vibrant",
+          "warm-hearted",
+          "witty",
+          "wondrous",
+          "zestful"
           ]
       noun <-
         [ "alpaca",
+          "armadillo",
+          "axolotl",
+          "badger",
           "blobfish",
+          "bobcat",
           "camel",
+          "capybara",
+          "caracal",
+          "cheetah",
+          "chameleon",
+          "chinchilla",
+          "chipmunk",
           "donkey",
+          "dormouse",
           "earwig",
+          "egret",
+          "elk",
           "ferret",
+          "fennec",
+          "fox",
+          "frog",
+          "gecko",
           "gerbil",
+          "gibbon",
+          "giraffe",
           "hamster",
+          "hedgehog",
+          "herron",
+          "hippo",
           "ibis",
           "jaguar",
+          "kangaroo",
+          "kiwi",
           "koala",
+          "ladybug",
           "lemur",
+          "leopard",
+          "lizard",
+          "llama",
+          "mallard",
           "marmot",
+          "mole",
+          "moonrat",
+          "moose",
+          "mouse",
           "narwhal",
+          "ocelot",
           "ostrich",
+          "otter",
+          "owl",
+          "panda",
+          "pangolin",
+          "penguin",
+          "platypus",
+          "polecat",
+          "porcupine",
+          "possum",
           "puffin",
           "quahog",
+          "racoon",
           "reindeer",
+          "rhino",
           "seahorse",
+          "seal",
+          "serval",
+          "shrew",
+          "sloth",
+          "starling",
+          "tapir",
+          "tiger",
+          "toad",
+          "toucan",
           "turkey",
+          "turtle",
           "urchin",
           "vole",
           "walrus",
+          "wallaby",
+          "wallaroo",
```

Comment:
These seem a little too similar to each other, but otherwise LGTM.

Reply:
I think it's ok since it's rare they that they will appear at the same time